.pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -x
+set -e
+
+make check-go-format
```

Comment:
That does not work well for me: 
````
+ set -e
+ make check-go-format
ERROR: These files differ from gofmt's style (run 'make format-go-code' to fix this):
./app/helper_functions.go
./bindata_assetfs.go
make: *** [check-go-format] Error 1
````
shouldn't we exclude the generated files from the `check-go-format` ?